account_statement_import_sheet_file/models/account_statement_import_sheet_mapping.py

```diff
@@ -95,10 +96,35 @@ class AccountStatementImportSheetMapping(models.Model):
             "transaction amount in original transaction currency from"
         ),
     )
+    amount_type = fields.Selection(
```

Comment:
These new fields are a bit misleading, and not sure if the default values is respecting how it was done before their introduction.

Reply:
This field is only used to display one or other fields in the mapping, avoiding the confusion of filling in some "not necessary" fields.

Reply:
But this is not what I'm saying. Is it the default value correct to display the same as in v15?

Reply:
Ok sorry, I understand. It is already added to the migration script to set the values of the previous records and keep the same functionality as in v15.

Reply:
Why added it to the migration script? Instead, the default should be the appropriate one

Reply:
This field is added in v16, and depending on what is selected the `amount_column`, `amount_debit_column`, `amount_credit_column` and `debit_credit_column` fields are shown or required (to avoid confusion).

In v16 there is no problem, and the default is not relevant in my opinion, but it is important that for the records that already exist the corresponding fields are shown, for that reason the migration script to set the correct value of `amount_type`.